recipes/opam/build.sh

```diff
@@ -0,0 +1,8 @@
+#!/bin/bash -e
+
+
+cp -r $SRC_DIR/* ${PREFIX}/bin/
```

Comment:
Since you need to build from source, I *think* the commands you want here are those at https://github.com/ocaml/opam/#compiling-without-ocaml, since `ocaml` won't be available.